NEWS.md

```diff
@@ -1,5 +1,6 @@
 # pkgdown (development version)
 
+* Bootstrap 3 has been deprecated. It was superseded in Dec 2021, and now we're starting to more directly encourage folks to move away from it.
```

Comment:
```suggestion
* Bootstrap 3 has been deprecated. It was superseded in December 2021, and now we're starting to more directly encourage folks to move away from it.
```

I always wonder how abbreviations are for screenreader users + people less at ease with English.